src/api.ts

```diff
@@ -65,15 +65,15 @@ export const callFnByName = (
 	let responseSent = false; // Flag to track if response has been sent
 	let errorCame = false;
 
-	cps[app].send({
+	childProcesses[app].send({
 		type: protocol.c,
 		fn: {
 			name,
 			args
 		}
 	});
 
-	cps[app].on('message', (data: childProcessResponse) => {
+	childProcesses[app].on('message', (data: childProcessResponse) => {
```

Comment:
```childProcesses``` resembles to childProcess and its hard to understand, use something different or lets keep it as it is.